truss/templates/server/model_wrapper.py

```diff
@@ -8,11 +9,10 @@
 from collections.abc import Generator
 from enum import Enum
 from pathlib import Path
-from queue import Queue
 from threading import Lock, Thread
-from typing import Any, Dict, Optional, Union
+from typing import Any, AsyncGenerator, Dict, Optional, Union
 
-from anyio import to_thread
+from anyio import Semaphore, to_thread
```

Comment:
Maybe a comment explaining why we use anyio.Semaphone vs asyncio.Semaphore. I guess because we're working with anyio threads, would be good to document.

Reply:
they are actually the same. I'm trying to standardize on `anyio` where we can.